docker-compose.yml

```diff
@@ -16,3 +18,5 @@ services:
       - "8000:8000"
     depends_on:
       - db
+    links:
+      - db:db
```

Comment:
thought this could fix the migrations issue I had, but it didn't. Ended up just running the app with `python manage.py runserver 8000`